rebasehelper/application.py

```diff
@@ -351,7 +351,7 @@ def patch_sources(self, sources):
         git_helper = GitHelper(sources[0])
         if not self.conf.non_interactive:
             git_helper.check_git_config()
-        patch = Patcher(self.conf.patchtool)
+        patch = Patcher("git")
```

Comment:
What do you think to define it globally.

Git command is already used here https://github.com/phracek/rebase-helper/blob/master/rebasehelper/utils.py#L490